src/hashmap.c

```diff
@@ -1,51 +1,36 @@
-#define BUCKETS 1000000
```

Comment:
imports should come before constant definitions. Same as in python and most other languages